src/backend/aspen/workflows/nextstrain_run/run_nextstrain_mpx.sh

```diff
@@ -74,11 +74,11 @@ $aws s3 cp /mpox/config/build_czge.yaml "${s3_prefix}/build_czge.yaml"
 $aws s3 cp /mpox/config/include.txt "${s3_prefix}/include.txt"
 
 # run snakemake, if run fails export the logs from snakemake to s3
-(cd /mpox && snakemake --printshellcmds --configfile config/build_czge.yaml --resources=mem_mb=312320) || { $aws s3 cp /mpox/.snakemake/log/ "${s3_prefix}/logs/snakemake/" --recursive ; $aws s3 cp /mpox/logs/ "${s3_prefix}/logs/mpox/" --recursive ; }
+(cd /mpox && snakemake --printshellcmds --configfile config/build_czge.yaml --resources=mem_mb=312320) || { $aws s3 cp /mpox/.snakemake/log/ "${s3_prefix}/logs/snakemake/" --recursive ; $aws s3 cp /mpox/results/mpxv/filter.log "${s3_prefix}/logs/mpox/" --recursive ; }
 
 # upload the tree to S3. The variable key is created to use later
 key="${key_prefix}/mpx_czge.json"
-$aws s3 cp /mpox/results/mpxv/tree.json "s3://${aspen_s3_db_bucket}/${key}"
+$aws s3 cp /mpox/auspice/monkeypox_mpxv.json "s3://${aspen_s3_db_bucket}/${key}"
```

Comment:
this is the final tree in `auspice` folder. tree name is specified [here](https://github.com/chanzuckerberg/czgenepi/blob/d9fe3677092c50a64fe6bdb3204f9f01da1d1bd0/src/backend/aspen/workflows/nextstrain_run/builds_templates/MPX_template.yaml#L18)